src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -48,13 +49,83 @@ bool change_pref_location(const std::string &input_path, const std::string &curr
     return true;
 }
 
+std::vector<std::string>::iterator modules;
+vfs::FileBuffer buffer;
+static ImGuiTextFilter search_bar;
+
 void draw_settings_dialog(HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     ImGui::Begin("Settings", &host.gui.configuration_menu.settings_dialog, ImGuiWindowFlags_AlwaysAutoResize);
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        if (vfs::read_file(VitaIoDevice::VS0, buffer, host.pref_path, "sys/external/libc.suprx")) {
+            ImGui::Text("Modules Listes\nSelect your desired modules.");
+            static const char *list_modules[] = {
+                // clang-format off
+                "activity_db", "adhoc_matching", "apputil", "apputil_ext", "audiocodec", "avcdec_for_player", "bgapputil", "bXCe",
```

Comment:
Put this into a "inc" file or something. I can walk you through on Discord later

Reply:
yes, i have thinks same but no have idea how set that hmm